test/isochrone.cc

```diff
@@ -90,9 +90,9 @@ void try_isochrone(loki_worker_t& loki_worker,
         rp("/features/" + std::to_string(i) + "/properties").Get(expected_response);
     EXPECT_TRUE((actual_properties && expected_properties) ||
                 (!actual_properties && !expected_properties));
-    if (expected_properties) {
-      ASSERT_EQ(actual_properties->GetObject(), expected_properties->GetObject());
```

Comment:
this is just temporary, but somehow gtest is _now_ complaining that this rapidjson type has no `operator==`, even though gtest and rapidjson didn't change. I wonder what it did before.. I'll rewrite this a bit.